lib/translation_maps/geo_resource_class.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
```

Comment:
Wowww thanks MatchMap!

Reply:
It's perfect for this! Bill clearly has done a lot of this type of thing, including in traject.